app/CustomCsp.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App;
+
+use Spatie\Csp\Directive;
+use Spatie\Csp\Policies\Basic;
+
+class CustomCsp extends Basic
+{
+    public function configure()
+    {
+        parent::configure();
+
+        $this
+            ->addDirective(Directive::BASE, 'self')
+            ->addDirective(Directive::STYLE, ['https://cloud.typography.com', 'https://spatie.be'])
+            ->addDirective(Directive::FONT, 'data:')
+            ->addDirective(Directive::SCRIPT, 'data:')
+            ->addDirective(Directive::CHILD, 'https://www.googletagmanager.com')
```

Comment:
We are using analytics on this site (through tag manager), could you set up the necessary directives for this as well? Take a look at murze.be for an example.